src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -147,6 +147,7 @@ internal static class IDEDiagnosticIds
 
         public const string UseNotPatternDiagnosticId = "IDE0083";
         public const string UseIsNotExpressionDiagnosticId = "IDE0084";
+        public const string SimplifyLinqExpressionDiagnosticId = "IDE0085";
```

Comment:
these gaps are intentional.  this group is for patterns.  This should go in the next major seciton.

Reply:
still applies :)